broker-cli/broker-daemon-client/index.js

```diff
@@ -80,7 +80,8 @@ class BrokerDaemonClient {
       if (certPathParts[0] === '~') {
         certPathParts[0] = os.homedir()
       }
-      const certPath = path.join(...certPathParts)
+
+      const certPath = path.normalize('/' + path.join(...certPathParts))
```

Comment:
might be worth trying if there is a way to get the root path without this leading slash.